src/main/java/seedu/trippie/Parser.java

```diff
@@ -1,16 +1,6 @@
 package seedu.trippie;
 
-import seedu.trippie.command.AddExpenseCommand;
-import seedu.trippie.command.AddPlaceCommand;
-import seedu.trippie.command.Command;
-import seedu.trippie.command.ExitCommand;
-import seedu.trippie.command.DeleteExpenditureCommand;
-import seedu.trippie.command.ListExpenseCommand;
-import seedu.trippie.command.BudgetCommand;
-import seedu.trippie.command.DeletePlaceCommand;
-import seedu.trippie.command.HelpCommand;
-import seedu.trippie.command.ListPlacesCommand;
-import seedu.trippie.command.SearchCommand;
+import seedu.trippie.command.*;
```

Comment:
It is preferred to mention explicitly which packages are imported

Reply:
okk